Legal.md

```diff
@@ -64,25 +64,18 @@ For instance, what if if(we) actually _were_ to create a non-profit foundation f
 
 Enter the [Developer's Certificate Of Origin](./starter_box/DCOO.md) (aka the DCOO).
 
-NOTE: ALL the below about the CLA needs to be replaced with information on the DCOO
-
-A CLA is just a specialized form of license. Just as the Apache License grants the right to use copyrighted software, a CLA is a license which grants the right for an individual or organization to use the work of the contributor. The contributor retains copyright over the work, but if(we) is allowed to distribute and administer the work as needed.
+The DCOO ensures that any contribution made to the project was created by the contributor and that it is not under any other copyright.  It also ensures that the contributor must certify that they are allowed to contribute the code and is under the appropriate open source license.
 
 **NOTA BENE**
-> Because of the potential legal and administrative snags involved, it is **absolutely vital** that all external contributors to your project have a signed CLA on file **before** you merge their contribution into the project.
-
-So, how do you do that?
-
-### The Contributor License Agreement Process
+> Because of the potential legal and administrative snags involved, it is **absolutely vital** that there is a pre-receive hook to ensure the Signed-Off-By is in place for any external contribution into the project.
 
-The process itself is quite simple:
+The process for the above is straightforward:
 
-1. Contributor sends in a pull request.
-1. You ask the contributor to send in a [signed CLA](./starter_box/CLA_Process.md).
-1. if(we) Legal files away the signed CLA then lets you know it's been received.
-1. You merge the pull request (after reviewing and approving it, naturally).
+1. Contributor sends in a pull request, with the commit(s) using a '--signoff' or '-s'.
```

Comment:
I think I would split this up:
1. Contributor creates commits using 'git commit --signoff' (or '-s' for short).
2. Contributor sends in a pull request.